examples/wildcard-binding-pattern/wildcard_binding_pattern.md

```diff
@@ -0,0 +1,10 @@
+# Wildcard binding pattern
+
+A wildcard binding pattern which is denoted by the `_` symbol, matches a value if the value belongs to type `any`. As long as the basic type of the value is not an `error`, it does not cause any assignment to be made to the variable.
```

Comment:
Let's add some usages of the `wildcard binding pattern` in the description. It is useful to ignore return values from a function, ignore values in a list